production/training/k8/alb-ingress-controller.yaml

```diff
@@ -0,0 +1,70 @@
+# Application Load Balancer (ALB) Ingress Controller Deployment Manifest.
+# This manifest details sensible defaults for deploying an ALB Ingress Controller.
+# GitHub: https://github.com/kubernetes-sigs/aws-alb-ingress-controller
```

Comment:
What is an "Ingress Controller"?

Reply:
It sets up external load balancing for EKS.  I will update the documentation soon.

https://aws.amazon.com/blogs/opensource/kubernetes-ingress-aws-alb-ingress-controller/